raft/raft.go

```diff
@@ -804,17 +804,16 @@ func (r *raft) campaign(t CampaignType) {
 		}
 		sort.Slice(ids, func(i, j int) bool { return ids[i] < ids[j] })
 	}
+	var ctx []byte
```

Comment:
thanks, look like the value of  variable doesn't changes in the loop, lgtm. /cc @tbg